Pinterest and Snapchat Audience Sync (#4836)
braze-inc/braze-docs: _docs/_hidden/private_betas/snapchat_audience_sync.md

---
nav_title: Audience Sync to Snapchat
article_title: Canvas Audience Sync to Snapchat

> description: "This reference article will cover how to use Braze Audience Sync to Facebook, to deliver advertisements based upon behavioral triggers, segmentation, and more."

Author: ```suggestion
description: "This reference article will cover how to use Braze Audience Sync to Snapchat, to deliver advertisements based upon behavioral triggers, segmentation, and more."
```

page_order: 4
alias: "/audience_sync_snapchat/"

Tool:
  - Canvas

> 

Author: ```suggestion
hidden: true
```

---

# Audience Sync to Snapchat

Using the Braze Audience Sync to Snapchat, brands can elect to add user data from their own Braze integration to Snapchat customer lists to deliver advertisements based on behavioral triggers, segmentation, and more. Any criteria you'd normally use to trigger a message (Push, Email, SMS, Webhook, etc.) in a Braze Canvas based on your user data can now be used to trigger an ad to that user in your Snapchat customer lists.

**Common use cases for audience syncing include:**

- Targeting high-value users via multiple channels to drive purchases or engagement
- Retargeting users who are less responsive to other marketing channels
- Creating suppression audiences to prevent users from receiving advertisements when they're already loyal consumers of your brand

> - Creating Lookalike Audiences to acquire new users more efficiently

Reviewer: ```suggestion
- Creating lookalike audiences to acquire new users more efficiently
```
similar audiences?

Once successfully connected, you will be taken back to the partner page, where you can view which accounts are connected and disconnect existing accounts.

![An updated version of the Snapchat technology partners page showing the ad accounts successfully connected.][3]{: style="max-width:80%;"}

Your Snapchat connection will be applied at the Braze app group level. If your Snapchat admin removes you from your Snapchat Business Manager or access to the connected Snapchat ad accounts, Braze will detect an invalid token. As a result, your active Canvases using Snapchat will show errors, and Braze will not be able to sync users.

### Step 2: Add an Audience Sync Step with Snapchat

Add a component in your Canvas and select **Audience Sync**.

![Workflow of the previous steps to add a Snapchat Audience component in Canvas Flow.][4]{: style="max-width:60%;"}

![Workflow of the previous steps to add a Snapchat Audience component in Canvas Flow.][5]{: style="max-width:60%;"}

### Step 3: Sync setup

Click on the **Custom Audience** button to open the component editor. Select Snapchat as the desired Audience Sync partner. 

> Then select your desired Snapchat ad account. Under the Choose a New or Existing Audience dropdown, type in the name of a new or existing audience.

Reviewer: ```suggestion
Then select your desired Snapchat ad account. Under the **Choose a New or Existing Audience** dropdown, type in the name of a new or existing audience.
```

![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account and existing audience are selected.]({% image_buster /assets/img/snapchat/snapchat6.png %})

{% endtab %}
{% endtabs %}

### Step 4: Launch Canvas

Once you have configured your Audience Sync to Snapchat, simply launch the Canvas! A new audience will be created, and users who flow through the Audience Sync step will be passed into this audience on Snapchat. If your Canvas contains subsequent components, your users will advance to the next step in their user journey.

You can view the audience in Snapchat by going into your ads manager account and selecting **Audiences** from the Assets section of the navigation. From the **Audiences** page, you can see the size of each audience after it reaches ~1,000.

![Audience details for a given Snapchat audience that includes audience name, audience type, audience size, and audience renetion in days.][9]

## User syncing and rate limit considerations

> As users reach the Audience Sync Step, Braze will sync these users in near real-time while also respecting Snapchat's API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Snapchat.

Reviewer: ```suggestion
As users reach the Audience Sync step, Braze will sync these users in near real-time while also respecting Snapchat's API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Snapchat.
```

> Snapchat's API rate limit states no more than ten queries per second and 100k users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.

Reviewer: ```suggestion
Snapchat's API rate limit states no more than ten queries per second and 100,000 users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.
```
